src/test/javascript/spec/component/code-editor/code-editor-actions.component.spec.ts

```diff
@@ -35,9 +32,9 @@ describe('CodeEditorActionsComponent', () => {
     let updateFilesStub: jest.SpyInstance;
     let commitStub: jest.SpyInstance;
 
-    beforeEach(async () => {
+    beforeEach(() => {
         return TestBed.configureTestingModule({
-            imports: [TranslateModule.forRoot(), ArtemisTestModule, AceEditorModule, FeatureToggleModule],
+            imports: [ArtemisTestModule, AceEditorModule, FeatureToggleModule],
```

Comment:
not sure if it fits for this test suite, maybe FeatureToggleModule can be removed here.. you can mock FeatureToggleDirective... In addition, you could also re-organize the assertions in this file. Some still uses toBeTruthy or toBeFalsy which are not recommended in guideline.

Reply:
I just checked, the directive itself cant be mocked, but the module can be replaced with only the directive